### ISSUE TITLE
MtlParser.GetMats() keep a reference of all materials

### ISSUE THREAD

**Issue by TrabacchinLuigi** [arcplus/ObjConvert#9]
This is a bad practice, should be the library consumer concern to keep a reference to the materials or to call the method once

**Comment by stoneflyop1** [comment]
You're right. It's a parser, should not keep materials' reference in it. I'll change this.

**Comment by TrabacchinLuigi** [comment]
if you want i'm already fixing lot of this issues in the fork i've made, but i'm also separating things, so for now i've dropped the support to tiles, for this reason i did not bother to make a pull request, but if you agree we could work there and when everything is well done merge it back here, or just have a look, i hope it will be at least inspiring
